spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -516,9 +520,9 @@
             expect(instance).to receive(:logger_info_hash).ordered.with(
               hash_including(
                 name: 'ThreatMetrix',
-                user_id: user.uuid,
-                threatmetrix_request_id: Proofing::Mock::DdpMockClient::TRANSACTION_ID,
+                threatmetrix_request_id: '1234',
```

Comment:
Wasn't this better before?

Reply:
I second this question. What was wrong with the `ddp-mock-transaction-id-123` value?

Reply:
After the refactoring it returns 1234 instead of Proofing::Mock::DdpMockClient::TRANSACTION_ID. Do we know if it should return Proofing::Mock::DdpMockClient::TRANSACTION_ID or some internal request ID that gets created?

Reply:
I feel like ideally we would change the source of this data instead of changing this test to match. (I don't think it is worth holding up the PR however.)